<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSelectorEdit</class>
 <widget class="QWidget" name="FileSelectorEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="pathEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="pathSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select ...</string>
     </property>
     <property name="icon">
      <iconset theme="folder"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pathOpen">
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset theme="system-run"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
